pyproject.toml

```diff
@@ -1,12 +1,77 @@
 [build-system]
-
 requires = [
-    "setuptools>=42",
-    "wheel"
+    "setuptools >= 59",
+    "wheel >= 0.41"
 ]
-
 build-backend = "setuptools.build_meta"
 
+[project]
+name = "odxtools"
+description = "Utilities to work with the ODX standard for automotive diagnostics"
+license = {text = "MIT"}
+authors = [
+    {name = "Katrin Bauer", email = "katrin.bauer@mbition.io"},
+    {name = "Andreas Lauser", email = "andreas.lauser@mbition.io"},
+    {name = "Ayoub Kaanich", email = "kayoub5@live.com"},
+]
+maintainers = [
+    {name = "Andreas Lauser", email = "andreas.lauser@mbition.io"},
+    {name = "Ayoub Kaanich", email = "kayoub5@live.com"},
+]
+readme = "README.md"
+requires-python = ">=3.8"
+keywords = ['can', 'can bus', 'DoIP', 'odx', 'pdx', 'obd', 'uds', 'automotive', 'diagnostics']
+classifiers = [
+    'Programming Language :: Python :: 3',
+    'Programming Language :: Python :: 3.8',
+    'Development Status :: 5 - Production/Stable',
+    'Environment :: Console',
+    'Intended Audience :: Developers',
+    'License :: OSI Approved :: MIT License',
+    'Operating System :: POSIX',
```

Comment:
Maybe use "OS Independent" for "Operating System", the library is pure python with no native dependencies

Reply:
ok, done. the `snoop` tool needs `python-can`, but I think `python-can` is available on most platforms and I don't consider the command line tools to be part of the core library...